docs/contributors/code/testing-overview.md

```diff
@@ -355,6 +355,18 @@ test( 'should render a darker background when isShady is true', () => {
 } );
 ```
 
+Similarly, the `toMatchStyleDiffSnapshot` function allows to snapshot only the difference between the _styles_ associated to two different states of a component, like in this example:
+
+```js
+test( 'should render margin', () => {
+	const { container: base } = render( <Spacer /> );
+	const { container: withMargin } = render( <Spacer margin={ 5 } /> );
+	expect( withMargin.firstChild ).toMatchStyleDiffSnapshot(
+		base.firstChild
+	);
```

Comment:
Nit: would it be okay to update to: 
```suggestion
	const { container: spacer } = render( <Spacer /> );
	const { container: spacerWithMargin } = render( <Spacer margin={ 5 } /> );
	expect( spacerWithMargin.firstChild ).toMatchStyleDiffSnapshot(
		spacer.firstChild
	);
```

It might be only me, but I like to read tests like sentences 😅